docs/developers-guide.md

```diff
@@ -24,24 +24,27 @@ If you have problems with your development environment, make sure that you are n
 
 These are the set of tools which are required in order to complete any build of the Metabase code. Follow the links to download and install them on your own before continuing.
 
-1. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - latest LTS version of JDK - more about [Java versions](./operations-guide/java-versions.md)
-2. [Node.js (http://nodejs.org/)](http://nodejs.org/) - latest LTS release
-3. [Yarn package manager for Node.js](https://yarnpkg.com/) - latest release of version 1.x
-4. [Leiningen (http://leiningen.org/)](http://leiningen.org/) - latest release
+1. [Clojure (ttps://clojure.org)](https://clojure.org/guides/getting_started) - install the latest release by following the guide depending on your OS
+2. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - you need to install JDK 8 (to build) and JDK 11 (to run Metabase) - more about [Java versions](./operations-guide/java-versions.md)
+3. [Node.js (http://nodejs.org/)](http://nodejs.org/) - latest LTS release
+4. [Yarn package manager for Node.js](https://yarnpkg.com/) - latest release of version 1.x - you can install it in any OS by doing `npm install --global yarn`
+5. [Leiningen (http://leiningen.org/)](http://leiningen.org/) - latest release
+6. [GetText package (https://www.gnu.org/software/gettext/)](https://www.gnu.org/software/gettext/) - latest
 
-On a most recent stable Ubuntu/Debian, the above tools can be installed by using:
+On a most recent stable Ubuntu/Debian, all the tools above, with the exception of Clojure, can be installed by using:
 
 ```
-sudo apt install openjdk-11-jdk nodejs yarnpkg leiningen
+sudo apt install gettext openjdk-11-jdk openjdk-8-jdk nodejs leiningen && sudo npm install --global yarn
```

Comment:
I generally advise against a special case of installation, such as the global install of Yarn (more detailed instructions on that is already available at yarn website), particularly when it's combined with system package (`apt` in this case).

Is there a reason why `yarnpkg` (from Apt) does not work for you?